hw/top_earlgrey/data/ip/chip_aon_timer_testplan.hjson

```diff
@@ -19,7 +19,10 @@
       lc_states: ["DEV", "PROD", "RMA"]
       features: ["AON_TIMER.WAKEUP.INTERRUPT"]
       tests: ["chip_sw_aon_timer_irq"]
-      bazel: ["//sw/device/tests:aon_timer_irq_test_cw310_test_rom"]
+      bazel: [
+        "//sw/device/tests:aon_timer_irq_test_fpga_cw310_test_rom",
+        "//sw/device/tests:aon_timer_irq_test_fpga_cw310_sival"
```

Comment:
Should all these use "...fpga_cw310_sival_rom_ext" instead?

Reply:
Yeah, done.